nvbench/printer_base.cuh

```diff
@@ -78,9 +78,9 @@ struct printer_base
 
   // move-only
   printer_base(const printer_base &)            = delete;
-  printer_base(printer_base &&)                 = default;
+  printer_base(printer_base &&)                 = delete;
```

Comment:
Hrm, it would be nice to keep the printers movable.

Does CI still complain if these `=default` statements are moved to the `printer_base.cxx` file, similar to the `printer_base` destructor?

If we need to delete these, I won't object, but the comment will need to be updated.

Reply:
Kept the move constructor and comment